launchpad-frontend/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "launchpad",
   "version": "0.1.0",
   "private": true,
-  "proxy": "http://localhost:5000",
+  "proxy": "http://localhost:5001",
```

Comment:
whyd u change this lol

Reply:
nikki told me to ;-; cuz it wasn't running properly she said sth about the ports on mac being different </3 sry

Reply:
leave this as 5000 for now we can discuss what port to change it to next standup.

Reply:
yez ma'am